Assets/HoloToolkit/Utilities/Editor/ProjectSettingsWindow.cs

```diff
@@ -0,0 +1,274 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+	/// <summary>
+	/// Renders the UI and handles update logic for HoloToolkit/Configure/Apply HoloLens Project Settings.
+	/// </summary>
+	public class ProjectSettingsWindow : AutoConfigureWindow<ProjectSettingsWindow.ProjectSetting>
+	{
+		// Nested Types
+		public enum ProjectSetting
+		{
+			ActiveBuildToWsa,
```

Comment:
I wonder if we should respect Unity's/Microsoft's casing here like:
1. Instead of Wsa to WSA
2. Uwp to UWP